modules/persistent-storage-local-install.adoc

```diff
@@ -19,6 +19,19 @@ The Local Storage Operator is not installed in {product-title} by default. Use t
 $ oc new-project local-storage
 ----
 
+. Optional: Add an annotation to the project that blocks a default node selector.
```

Comment:
I would make the title of this step a bit more descriptive to the user's end goal. "Default node selector" is an implementation detail, and doesn't answer the "why" behind this step. What about something like:

> Optional: Allow local storage to be created on infrastructure nodes.